server/events/markdown_renderer.go

```diff
@@ -58,8 +58,8 @@ type MarkdownRenderer struct {
 	executableName           string
 }
 
-// commonData is data that all responses have.
-type commonData struct {
+// CommonData is data that all responses have.
+type CommonData struct {
```

Comment:
Does this need to expose to public? `CommonData` is used only in markdown_rendered.go, so `commonData` is better instead `CommonData`. Others too.